packages/components/psammead-navigation/src/index.jsx

```diff
@@ -227,7 +227,9 @@ const StyledNav = styled.nav`
     ampOpenClass &&
     css`
       &.${ampOpenClass} {
```

Comment:
Has this `&.${ampOpenClass} {` been put back in again? I thought it worked without it https://github.com/bbc/psammead/pull/2785#discussion_r355326357

Reply:
We realised we do need it. Because it's amp, we have to do things based on classes rather than the props, so `ampOpenClass` always gets passed in by Simorgh (in the SSR) and then when pressing the menu button, amp toggles the `ampOpenClass` on and off on the navigation bar.

Reply:
Ok, I have missed this comment https://github.com/bbc/psammead/pull/2785#issuecomment-563212707 👍